netlify-plugin-cloudinary/src/index.ts

```diff
@@ -236,15 +233,18 @@ export async function onBuild({
     await Promise.all(
       Object.keys(_cloudinaryAssets).flatMap(mediaType => {
         // @ts-expect-error what are the expected mediaTypes that will be stored in _cloudinaryAssets
-        return _cloudinaryAssets[mediaType].map(async asset => {
-          const { publishPath, cloudinaryUrl } = asset;
-          netlifyConfig.redirects.unshift({
-            from: `${publishPath}*`,
-            to: cloudinaryUrl,
-            status: 302,
-            force: true,
+        if (Object.hasOwn(_cloudinaryAssets[mediaType], 'map')) {
```

Comment:
just curious about the context of this change - did you hit an additional issue where you needed to add this line? seems like it was compiling before which is why i ask